src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
 
 Moreover, it is also important to notice that the Scheduler and 3 microservices (*scheduling-api*, *job-planner*, and *proactive-cloud-watch*) have to point to the same scheduler database since they use that one.
 
+==== Example configuration with PostgreSQL
+It is recommanded to use a PostgreSQL database in a ProActive prod environment.
+In the following, we show an example of PostgreSQL installation and configuration to be used along with ProActive.
+
+===== PostgreSQL version
+PostgreSQL version 12.2 or 12.9 is required to be installed. PostgreSQL could be installed on the same machine as for ProActive or on a separate server (on-prem or on the cloud).
+
+===== Initialize the database
+To initialize the DB, some SQL queries should be executed in the PostgreSQL server machine.
+To do so, please create the init script in the /tmp folder for example and put the following SQL queries:
```

Comment:
In order to do so, the following SQL queries can be defined in a SQL initialization script: